src/action/filesystems.rs

```diff
@@ -12,6 +12,53 @@
 use rrg_proto::{Filesystem, KeyValue, AttributedDict, DataBlob};
 use crate::session::{self, Session};
 
+use log::error;
+use std::fmt::{Display, Formatter};
+
+/// Enum of possible errors, which can occur during collecting filesystems data.
+#[derive(Debug)]
+enum Error {
+    /// Missed mtab-like file error.
+    MissedFile(std::io::Error),
+    /// Parsing mtab-like file error.
+    MountInfoError(std::io::Error),
```

Comment:
I replaced this variant name with `MountInfoParse`. I don't think we should name it `MountInfo`, because of presence of `MountInfo` in `proc_mounts` crate, even it's in another scope.

Reply:
I think your solution makes sense, marking as resolved.